modules/services/podman-linux/podman-lib.nix

```
@@ -83,8 +85,35 @@ in {
         else
           [ ];
 
+      checkImageTag = extraConfig:
+        let
+          buildSection = if (builtins.hasAttr "Build" extraConfig) then
+            extraConfig.Build
+          else
+            { };
+          imageTags = if (builtins.hasAttr "ImageTag" buildSection) then
+            buildSection.ImageTag
+          else
+            [ ];
```

Perhaps this could be simplified to
```suggestion
          imageTags = (extraConfig.Build or {}).ImageTag or [];
```